src/content/docs/apis/rest-api-v2/account-examples-v2/listing-users-your-account.mdx

```diff
@@ -12,15 +12,20 @@ redirects:
   - /docs/apis/rest-api-v2/requirements/listing-users-your-account
 ---
 
-New Relic stores the list of individuals who can use your account in a database by their [email address](/docs/accounts-partnerships/accounts/account-setup/adding-updating-users), assigned [role](/docs/accounts-partnerships/accounts/account-setup/users-roles), and their first and last name if provided. You can view this data from New Relic's [user interface](/docs/accounts-partnerships/accounts/account-setup/adding-updating-users) and from the [API Explorer (v2)](/docs/apm/apis/api-explorer-v2/parts-api-explorer).
+For New Relic users on our [original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models), we store a list of the users who can access your account in a database by their [email address](/docs/accounts-partnerships/accounts/account-setup/adding-updating-users), assigned [role](/docs/accounts-partnerships/accounts/account-setup/users-roles), and their first and last name if provided. You can view this data from New Relic's [user interface](/docs/accounts-partnerships/accounts/account-setup/adding-updating-users) and from the [API Explorer (v2)](/docs/apm/apis/api-explorer-v2/parts-api-explorer).
 
 <Callout variant="tip">
   To obtain the same information from the [New Relic API Explorer (v2)](https://rpm.newrelic.com/api/explore), select **Users > GET List**.
 </Callout>
 
-## Listing all account users [#list_all_users]
+## Requirements [#requirements]
```

Comment:
The structure of this section confuses me a bit. Having a header called "Requirements" followed by a single line that says "Requirements" throws me off. No real suggestions, but there may be a better way to structure this.